command/operator_diagnose_test.go

```diff
@@ -41,130 +41,130 @@ func TestOperatorDiagnoseCommand_Run(t *testing.T) {
 			},
 			[]*diagnose.Result{
 				{
-					Name:   "Parse configuration",
+					Name:   "Parse Configuration",
 					Status: diagnose.OkStatus,
 				},
 				{
-					Name:   "init-listeners",
+					Name:   "Start Listeners",
 					Status: diagnose.WarningStatus,
 					Children: []*diagnose.Result{
 						{
-							Name:   "create-listeners",
+							Name:   "Create Listeners",
 							Status: diagnose.OkStatus,
 						},
 						{
-							Name:   "check-listener-tls",
+							Name:   "Listener TLS Checks",
```

Comment:
Noun to verb suggestion: "Check Listener TLS" or "Check TLS Listeners". 

I'm adopting this pattern for all the names that are phrased similarly. The alternative for these is "Perform [text] Checks"